engine/runtime-compiler/src/main/java/org/enso/compiler/pass/desugar/OperatorToFunctionMini.java

```diff
@@ -17,13 +17,14 @@ public Expression transformExpression(Expression ir) {
       ListBuffer<CallArgument> args = new ListBuffer<>();
       args.addOne(binOp.left());
       args.addOne(binOp.right());
-      return new Application.Prefix(
```

Comment:
With a change like this the `new Application.Prefix` constructor can be made package private.

Reply:
Cannot be done unfortunatelly, as PersistableProcessor requires a public constructor. If `Application.Prefix` ctor is package-private, there will be failure in [PersistableProcdessor.generatePersistance](https://github.com/enso-org/enso/blob/3e730e9569cd3692c3acb35b35f62e48e5b18dee/lib/java/persistance-dsl/src/main/java/org/enso/persist/impl/PersistableProcessor.java#L201)